### DISCUSSION TITLE
viewing distance

### DISCUSSION THREAD

**Discussion by farfromrefug** [visgl/deck.gl#5650] [Q&A] [answered]
Is there a way to set viewing distance?
<img width="1217" alt="Screenshot 2021-04-10 at 18 19 57" src="https://user-images.githubusercontent.com/655344/114276970-5e380b80-9a29-11eb-83cd-7dd98940da03.png">
There i would like to reduce the loading load by reducing the "viewing distance". 
I have looked at `far` but it does not seem to be doing what i want

On the same "subject" is there a way to define if a text,icon, feature is to be shown based on its distance to camera/position?

**Comment by Pessimistress** [comment, marked as answer]
In MapView, `zoom` controls the distance between the camera and the target.

**Reply by farfromrefug** [reply]
@Pessimistress sorry but i dont think i understand. 
I know that in ViewState zoom defines the tile zoom used to render tiles.
I have look at `MapView` and `MapController` and the only `zoom` i see is the tile zoom level.
Can you point me to the code your are talking about?

In `FirstPersonView` the `far` option is working as i expect it to and i can limit the `viewing` distance with it. 
But it does not seem to be doing anything in `MapView`

**Reply by Pessimistress** [reply]
It is not clear to me what you are referring to by "viewing distance." If you are looking for the `far` equivalent in MapView it is `farZMultiplier`: https://deck.gl/docs/api-reference/core/map-view#constructor-1

**Reply by farfromrefug** [reply]
@Pessimistress that s it! Sorry if it wasnt easy to understand. With `farZMultiplier` i can change what i call the "viewing distance" .
